### PR TITLE
Ana/4746 add LiBalance component

### LINKED ISSUE #4746: convert TmBalance row into a reusable component
so we can add it to the bottom section of the portfolio page

_Originally posted by @jbibla in https://github.com/luniehq/lunie/pull/4745#issuecomment-676661550_

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4746 

**Description:**

Adds the LiBalance component

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f3eacb966b50a018ce6e305--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
I don't know why but the tests for LiBalance are not working :shrug: not sure what I'm doing wrong

File: app/src/components/common/BalanceRow.vue [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+      </div>
+    </template>
+  </div>
+</template>
+<script>
+import { bigFigureOrShortDecimals } from "scripts/num"
+import { fromNow } from "src/filters"
+import { mapGetters, mapState } from "vuex"
+export default {
+  name: `balance-row`,
```
[the remark this answers is not in the document]
Comment (jbibla) [reply app/src/components/common/BalanceRow.vue:75]: `<li></li>`

if LiValidator isn't that component then it is also a bad name 😄

File: app/src/components/staking/Undelegations.vue [hunk 2]
```diff
@@ -31,13 +36,25 @@ export default {
...
-    ...mapGetters([`address`, `network`]),
+    ...mapGetters([`address`, `network`, `currentNetwork`]),
+  },
+  methods: {
+    convertUndelegationsToBalances(undelegations) {
+      return undelegations.map((undelegation) => {
+        return {
+          ...undelegation,
+          total: undelegation.amount,
+          denom: this.currentNetwork.stakingDenom,
```
[the remark this answers is not in the document]
Comment (jbibla) [reply app/src/components/staking/Undelegations.vue:56]: seems like either the schema should be updated to account for substrate chains or we should have a new schema - that i'm not sure about - but doing data manipulation on the FE is no bueno. ok for now though, we can merge and refactor.

**Comment by Bitcoinera** [comment]
Actually the E2E earlier failed due to this, nothing that this PR changed:

**Comment by Bitcoinera** [comment]
@jbibla let's merge! :woman_dancing:

File: app/src/components/common/BalanceRow.vue [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+      <div
+        v-if="balance.endTime"
+        :key="balance.denom + '_endtime'"
+        class="table-cell endtime"
+      >
+        <span>
+          {{ balance.endTime | fromNow }}
+        </span>
+      </div>
+    </template>
```
Comment (Bitcoinera) [inline app/src/components/common/BalanceRow.vue:14]: This is broken:

image


Please, test before pushing

**Comment by Bitcoinera** [comment]
image

:eyes:

**Comment by Bitcoinera** [comment]
OK, now it is good

**Comment by Bitcoinera** [comment]
We should this merge this one next so we can use this component for the undelegations and bonded stake

**Pull request merged**